packages/solana-connector/package.json

```
@@ -24,7 +24,7 @@
   "peerDependencies": {
     "@web3modal/core": ">=5.0.0",
     "@web3modal/scaffold": ">=5.0.0",
-    "fuels": ">=0.94.6"
+    "fuels": ">=0.94.98"
```

there's a typo everywhere here.

```suggestion
    "fuels": ">=0.94.8"
```

---

Nice catch, thanks